### DISCUSSION TITLE
Compressor on-off cyle

### DISCUSSION THREAD

**Discussion by piplica** [gekkekoe/esphome-ecodan-hp#25] [Ideas]
Pls add this also, we have working hour, which is great, but this is also important feature.
Thank U.

**Comment by bwduncan** [comment]
It's not clear what you want. Please be specific.

Is there some other software which does this? Is it a feature in MELCloud?

You can see when the compressor starts and stops by looking at the compressor frequency sensor.

**Reply by piplica** [reply]
We can see total working hours, also from ecodan_heatpump_operating_runtime, but from the FTC U can also read total starts from compressor, which is very Useful info.

**Comment by gekkekoe** [comment]
there's no cmd known to do that over cn105. Also the procon cannot do it.In general if the procon can do it, it can be reversed.

the FTC controller is different. having said that, you could create a sensor that counts when compressor frequency is not 0, but it won't give you the total run hours (lifetime)
